scripts/update_and_rebuild_petsc.sh

```diff
@@ -96,10 +102,8 @@ if [ -z "$go_fast" ]; then
       --F77FLAGS='-fPIC -fopenmp' \
       --with-cxx-dialect=C++11 \
       --with-fortran-bindings=0 \
-      --with-sowing=0 \
-      $* \
-
-   make all
-else
-   make all
```

Comment:
Looks like you may have unintentionally broke the "--fast" option? This option allows developers who know what they are doing the ability to run this script without re-running configure (i.e. just make and make install). I've never used this option so I'm not sure if it works with PETSc - it should. Fande copied it from the libMesh script. @fdkong is this OK to delete? looks like add the "-j" option is a nice improvement.

Reply:
I put the make syntax outside of the if statement containing the configuration (line 108), so --fast should just skip the if block and go directly to running make.

Reply:
Oh yes, no difference in the two branches. 👍

Reply:
We should keep these. I can use "--fast" to skip configuration if I want.

Reply:
I see